README.md

```diff
@@ -104,10 +104,11 @@ $ rubycritic --help
 Analyzer Configuration
 -------------------------
 
-* `Reek`: `RubyCritic` just uses `Reek`'s default [configuration loading mechanism](https://github.com/troessner/reek#configuration-file).
-
-This section is work in progress at the moment and is missing the other analyzers that RubyCritic supports.
-You can expect this to be updated soon.
+* [`Reek`](https://github.com/troessner/reek): `RubyCritic` utilizes `Reek`'s default [configuration loading mechanism](https://github.com/troessner/reek#configuration-file).
+  This means that if you have an existing `Reek` configuration file, you can just put this into your
+  project root and `RubyCritic` will respect this configuration.
+* [`flay`](https://github.com/seattlerb/flay): We use `flay`'s default configuration.
+* [`flog`](https://github.com/seattlerb/flog): We use `flog`'s default configuration with a couple of [smaller tweaks](https://github.com/whitesmith/rubycritic/blob/master/lib/rubycritic/analysers/helpers/flog.rb#L5).
```

Comment:
> BTW this flog configuration behaviour is nowhere spec'ed.

It's kinda spec'ed on the [CLI code](https://github.com/seattlerb/flog/blob/258a1c5f0e3b356869804e47bd7c7c400a1488b6/lib/flog_cli.rb#L76).

Anyway, here are explanations for the options:

* `all` - Forces Flog to report scores on all classes and methods. Without this option Flog will only give results up to a certain threshold.

* `continue` - Makes it so that Flog does not abort when a ruby file cannot be parsed. I wish RubyCritic had this option so it could be properly delegated to the other gems.

* `methods` - Configures Flog to skip code outside of methods. It prevents Flog from reporting on the "methods" `private` and `protected`. It also prevents Flog from reporting on Rails methods like `before_action` and `has_many`.